fixtures/cameo/zenit-z120.json

```diff
@@ -0,0 +1,270 @@
+          "type": "ColorPreset",
+          "comment": "red"
+        },
+        {
+          "dmxRange": [14, 21],
+          "type": "ColorPreset",
+          "comment": "amber"
+        },
+        {
+          "dmxRange": [22, 29],
+          "type": "ColorPreset",
+          "comment": "yellow warm"
+        },
+        {
+          "dmxRange": [30, 37],
+          "type": "ColorPreset",
+          "comment": "yellow"
+        },
+        {
+          "dmxRange": [38, 45],
+          "type": "ColorPreset",
+          "comment": "green"
+        },
+        {
+          "dmxRange": [46, 53],
+          "type": "ColorPreset",
+          "comment": "turquoise"
+        },
+        {
+          "dmxRange": [54, 61],
+          "type": "ColorPreset",
+          "comment": "cyan"
+        },
+        {
+          "dmxRange": [62, 69],
+          "type": "ColorPreset",
+          "comment": "blue"
+        },
+        {
+          "dmxRange": [70, 77],
+          "type": "ColorPreset",
+          "comment": "lavender"
+        },
+        {
+          "dmxRange": [78, 85],
+          "type": "ColorPreset",
+          "comment": "mauve"
+        },
+        {
+          "dmxRange": [86, 93],
+          "type": "ColorPreset",
+          "comment": "magenta"
+        },
+        {
+          "dmxRange": [94, 101],
+          "type": "ColorPreset",
+          "comment": "pink"
+        },
+        {
+          "dmxRange": [102, 109],
+          "type": "ColorPreset",
+          "comment": "warm white"
+        },
+        {
+          "dmxRange": [110, 117],
+          "type": "ColorPreset",
+          "comment": "white"
+        },
+        {
+          "dmxRange": [118, 125],
+          "type": "ColorPreset",
+          "comment": "cold white"
+        },
+        {
+          "dmxRange": [126, 127],
+          "type": "ColorPreset",
+          "comment": "color jumping stop"
+        },
+        {
+          "dmxRange": [128, 191],
+          "type": "ColorPreset",
+          "comment": "color jumping speed slow fast"
+        },
+        {
+          "dmxRange": [192, 255],
+          "type": "ColorPreset",
+          "comment": "colour fading speed slow fast"
+        }
+      ]
+    },
+    "Zoom": {
+      "capability": {
+        "type": "Zoom",
+        "angleStart": "narrow",
+        "angleEnd": "wide"
+      }
+    },
+    "Settings": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 5],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [6, 127],
+          "type": "Maintenance",
+          "hold": "5s",
+          "comment": "dimmer response LED"
+        },
+        {
+          "dmxRange": [128, 191],
+          "type": "Maintenance",
+          "comment": "dimmer response HALOGEN"
+        },
+        {
+          "dmxRange": [192, 255],
+          "type": "Maintenance",
+          "hold": "5s",
+          "comment": "reset all fonctions"
+        }
+      ]
+    }
+  },
+  "modes": [
```

Comment:
There's also 3ch, 4ch, 5ch, 10ch, 11ch, and 17ch modes, but they're mostly just remixes of the channels which are already there:

```
{
  "name": "3 channel",
  "shortName": "3ch",
  "channels": [
    "Dimmer",
    "Color Temperature",
    "Zoom"
  ]
},
{
  "name": "4 channel",
  "shortName": "4ch",
  "channels": [
    "Dimmer",
    "Strobe",
    "Color Macros",
    "Zoom"
  ]
},
{
  "name": "5 channel",
  "shortName": "5ch",
  "channels": [
    "Red",
    "Green",
    "Blue",
    "White",
    "Zoom"
  ]
}
```

```
{
  "name": "10 channel",
  "shortName": "10ch",
  "channels": [
    "Red",
    "Red fine",
    "Green",
    "Green fine",
    "Blue",
    "Blue fine",
    "White",
    "White fine",
    "Zoom",
    "Zoom fine"
  ]
},
{
  "name": "11 channel",
  "shortName": "11ch",
  "channels": [
    "Dimmer",
    "Strobe",
    "Red",
    "Green",
    "Blue",
    "White",
    "Color Macros",
    "Color Temperature",
    "Zoom",
    "Dimmer Curve",
    "Settings",
  ]
},
{
  "name": "17 channel",
  "shortName": "17ch",
  "channels": [
    "Dimmer",
    "Dimmer fine",
    "Strobe",
    "Red",
    "Red fine",
    "Green",
    "Green fine",
    "Blue",
    "Blue fine",
    "White",
    "White fine",
    "Color Macros",
    "Color Temperature",
    "Zoom",
    "Zoom fine",
    "Dimmer Curve",
    "Settings",
  ]
},
```

New channels needed: fine channels (R/G/B/W/Z), "Color Temperature", "Dimmer Curve".